CHANGELOG.md

```diff
@@ -18,8 +18,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
       of Origin](https://developercertificate.org). For details, see
       [Contributing to Open Enclave](docs/Contributing.md).
     - The copyright for all sources is now attributed to Open Enclave SDK contributors.
-- Update Intel DCAP library dependencies to 1.3.1.
-- Update Intel PSW dependencies to 2.5.101.3 on Windows.
+- Update Intel DCAP library dependencies to 1.4.1.
+- Update Intel PSW dependencies to 2.6.100.2 on Windows.
+- Update Intel SGX depencies to accomodiate sgx_enclave_common.h split
```

Comment:
Typo: "dependencies" and "accommodate the"

What does the sgx_enclave_common.h split refer to though? It's probably useful to elaborate if the dev needs to understand something different about how they need to setup their environment. Otherwise if its purely something we had to do internally to absorb the change, then it doesn't need to be mentioned in the CHANGELOG.md.

Reply:
Removed, thanks.